benchmarks/coin_state_streaming.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+import asyncio
+from time import monotonic
+
+from benchmarks.utils import setup_db
+from chia.full_node.coin_store import CoinStore
+from chia.full_node.hint_store import HintStore
+from chia.util.ints import uint32
+from tests.core.full_node.stores.test_coin_store import generate_crs, generate_phs
+
+
+async def run_benchmark(version: int) -> None:
```

Comment:
would it be very inconvenient to move this into `test_coin_store.py` and mark it as a "benchmark"?
Currently we run tests with `-m "not benchmark"` in CI, and then we run (serially) only the benchmarks.

There is some precedence for adding benchmarks this way, especially in measuring the mempool.
If the test takes 10 minutes, it might not be great to run it in CI however.

I think @altendky may have some plans for the `benchmarks` directory.